docs/speech-usage.rst

```diff
@@ -83,5 +83,73 @@ data to text and returns alternative text transcriptons.
      transcript: Hello, this is one test
      confidence: 0
 
+
+Streaming Recognition
+---------------------
+
+The :meth:`~google.cloud.speech.Client.stream_recognize` method converts speech
+data to possible text alternatives on the fly.
+
+.. note::
+    Streaming recognition requests are limited to 1 minute of audio.
+
+    See: https://cloud.google.com/speech/limits#content
+
+.. code-block:: python
+
+    >>> from google.cloud import speech
+    >>> client = speech.Client()
+    >>> results = client.stream_recognize('hello.flac', 'FLAC', 44100)
+    >>> results[0].is_final
+    False
+    >>> results[2].is_final
+    True
+    >>> results[2].alternatives[0].transcript
+    hello
+    >>> results[2].alternatives[0].confidence
+    0.96976006031
+
+For continuous speech containing more than one word, the ``single_utterance``
+option should be disabled.
+
+See: `Single Utterance`_
+
+.. code-block:: python
+
+    >>> results = client.stream_recognize('hello.flac', 'FLAC', 44100,
+    ...                                   single_utterance=False)
+
+
+If you would like interim results to be returned as well as the final results,
+you can set the ``interim_results`` option to ``True``.
+
+  .. code-block:: python
+
+    >>> results = client.stream_recognize('hello.flac', 'FLAC', 44100,
+    ...                                   interim_results=True)
+    >>> print results
+    [{
+        alternatives: {
+            transcript: 'how'
+        },
+        stability: 0.00999999977648
+    },
+    {
+        alternatives: {
+            transcript: 'hello'
+        }
+        stability: 0.00999999977648
+    },
+    {
+        alternatives: {
+            transcript: 'hello'
+            confidence: 0.96976006031
+        }
+        is_final: true
+    }
+    ]
```

Comment:
Yeah so, the GAPIC call, with `interim_results=True` returns something like this...

```bash
[alternatives {
  transcript: "how"
}
stability: 0.00999999977648
, alternatives {
  transcript: "hello"
}
stability: 0.00999999977648
, alternatives {
  transcript: "hello"
  confidence: 0.96976006031
}
is_final: true
]
```

I took that out in a later version of this doc though. I'm not sure why you're seeing it?